### ISSUE TITLE
Pepsinogen I low in blood

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11650]
**Preferred term label:**
Pepsinogen I low in blood

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C1695692

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Atrophic gastritis  https://www.ncbi.nlm.nih.gov/books/NBK563275/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001531
- label: Decreased circulating pepsinogen I concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: PMID:26556485
- def: The concentration of pepsinogen I in the blood circulation is below the lower limit of normal.
- comment: Human pepsinogens are proenzymes for pepsin, a digestive enzyme produced by gastric chief cells. Human pepsinogens are biochemically and immunochemically classified into two groups: pepsinogen I and pepsinogen II. Serum pepsinogen (SPG) may function as a marker of the functional and morphologic status of the gastric mucosa, including atrophic changes and inflammation, such as H. pylori infection, AG and IM. Serum PGI and PGII levels are increased with increasing severity of H. pylori-related chronic gastritis. However, when atrophic changes in the corpus are accompanied by a loss of cells in the corpus, including those secreting PGI, the level of PGI decreases, whereas the level of PGII remains high or stable.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11650
- synonyms:
